src/marketplace/ListingsView.js

```diff
@@ -196,26 +199,40 @@ class ListingsView extends React.Component<Props, State> {
 			response => response.json().then(data => {
 				if (response.status !== 201) {
 					console.log(response.status);
-					console.log(data.msg);
+					toast(data.msg);
```

Comment:
I'm still kind of on the fence about this since iirc toasts go away automatically so this might be a little annoying to debug, but if we are going to do it, at least use toast.error() for these: https://github.com/fkhadra/react-toastify#usage